README.md

```diff
@@ -766,6 +766,111 @@ The `/api/v1/export` endpoint should return the following response:
 Extra labels may be added to all the imported time series by passing `extra_label=name=value` query args.
 For example, `/api/put?extra_label=foo=bar` would add `{foo="bar"}` label to all the ingested metrics.
 
+## How to send data from NewRelic infrastructure agent
+
+VictoriaMetrics accepts data from NewRelic infrastructure agent API at `/api/v1/newrelic/infra/v2/metrics/events/bulk` path.
+NewRelic's infrastructure agent sends so-called [Events](https://docs.newrelic.com/docs/data-apis/understand-data/new-relic-data-types/#event-data)
+which are transformed to the [Prometheus exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format).
+
+By default, NewRelic's infrastructure agent sends next types of the infrastructure events:
+1. SystemSample
+2. ProcessSample
+3. StorageSample
+4. NetworkSample
+5. ContainerSample
+6. InfrastructureEvent
+Those all events processed by VictoriaMetrics and prepare the [Prometheus exposition format](https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-based-format).
+
+### Sending metrics to VictoriaMetrics
+
+NewRelic's infrastructure agent allows configuring destinations for metrics sending via ENV variable `COLLECTOR_URL`.
```

Comment:
Can I used empty license key or use random value?

Reply:
Can you have multiple destinations for metrics delivery configured for one agent?

Reply:
> Can I used empty license key or use random value?

if the license key is empty agent will show an error 
```
load configuration file" component="New Relic Infrastructure Agent" error="no license key, please add it to agent's config file or NRIA_LICENSE_KEY environment variable"
```
if we use a randomly generated key we will get 400 error 
```
level=warning msg="agent connect attempt failed" component=IdentityConnectService error="ingest rejected connect: 401 401 Unauthorized "
```
I mentioned in the documentation that it is required
```
It is required to specify `NRIA_LICENSE_KEY` which can get only after registration into account of the NewRelic cloud.
```

Reply:
> Can you have multiple destinations for metrics delivery configured for one agent?

COLLECTOR_URL env flag supports only one destination for metrics delivery. Even if we try to run 
```COLLECTOR_URL="http://localhost:8428/api/v1/newrelic" COLLECTOR_URL="http://localhost:8429/api/v1/newrelic" NRIA_LICENSE_KEY="3a6d9628dc0ae4f7301525afc0da9cce1419NRAL"  ./newrelic-infra```
the agent will take the last defined COLLECTOR_URL